### PR TITLE
User resource Implementation with dao

### PR DESCRIPTION
**Pull request by nnenna352**

### CODE REVIEW

File: src/main/java/firebase/sdk/dashboard/dao/UserDaoDatastore.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+
+
+
+
+
+
+
+
+}
```
Comment (ankitaj224) [inline src/main/java/firebase/sdk/dashboard/dao/UserDaoDatastore.java]: Why?

File: src/main/java/firebase/sdk/dashboard/api/UserResource.java [hunk 2]
```diff
@@ -41,8 +38,8 @@ public UserResource() {
   @GET
   @Path("{uid}/sdks")
   public Response getUserFavorites(@PathParam("uid") String uid) {
-    // TODO: implement this method
     return ResponseHandler.createJsonResponse(Status.OK, favourites);
+
```
Comment (shrutip90) [inline src/main/java/firebase/sdk/dashboard/api/UserResource.java]: Remove blank line.

File: src/main/java/firebase/sdk/dashboard/dao/UserDao.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  /**
+   * Registers the changes in the User object to the database.
+   *
+   * @param user The user being updated in the data source.
+   */
+  public void updateUser(User user);
+  
+}
+ 
+
```
Comment (shrutip90) [inline src/main/java/firebase/sdk/dashboard/dao/UserDao.java]: Extra blank line.

File: src/main/java/firebase/sdk/dashboard/dao/UserDaoDatastore.java [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        FilterPredicate userPropertyFilter = makePropertyFilter("id", id);
+        Query query = new Query("User").setFilter(userPropertyFilter);
+
+        PreparedQuery preparedQuery = DATASTORE.prepare(query);
+        FetchOptions fetchOptions = FetchOptions.Builder.withDefaults();
+        Entity user = preparedQuery.asSingleEntity();
+        if (user == null) {
+            return null;
+        }
+
```
Comment (shrutip90) [inline src/main/java/firebase/sdk/dashboard/dao/UserDaoDatastore.java]: Remove blank line.

**Review by ankitaj224 — APPROVED** [review]

**Pull request merged**